wordpress/user.py

```diff
@@ -17,9 +17,12 @@ def fetch(self, email):
         body = self.client.get_request(f'users/?search={email}')
         if len(body):
             user_id = body[0]['id']
-            body = self.client.get_request(f'users/{user_id}', "context=edit")
-            self._convert_to_resource(body)
+            self.fetch_by_email(user_id)
+        return self
 
+    def fetch_by_email(self, user_id):
```

Comment:
:warning: it should be `fetch_by_email`

Reply:
It was changed in the last PR. Maybe branch was not updated.

Reply:
You mean probably fetch_by_id and the previous to fetch_by_email isn't ?